qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/ManageLocationTagsPage.java

```diff
@@ -84,4 +85,8 @@ public void cancelRetireLocation() {
 		waitForElement(CONFIRM_RETIRE_LOCATION_BUTTON);
 		clickOn(CONFIRM_RETIRE_LOCATION_BUTTON);
 	}
+
+	public void returnToConfigureMetaDataHomePage() {
```

Comment:
```suggestion
	public void returnToConfigureMetaDataPage() {
```

Reply:
> 

@sherrif10 there is an already exisiting method with such a name

Reply:
Which method is that.

Reply:
Do you mind addressing this comment here.

Reply:
> Which method is that.

@sherrif10  this https://github.com/openmrs/openmrs-contrib-qaframework/blob/25dedb1043e0bac74946b8cf35b937d002d59734/qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/ManageLocationAttributeTypesPage.java#L120